doc/design.rst

```diff
@@ -158,7 +158,7 @@ A Pack's structure is as follows:
 
 ::
 
-    EncryptedBlob1 || ... || EncryptedBlobN || EncryptedHeader || Header_Length
+    EncryptedBlob1 || ... || EncryptedBlobN || Random bytes padding || EncryptedHeader || Header_Length
```

Comment:
This breaks the file size check in `prune` and `check` and therefore isn't an option.